bin/block_generator/src/main.cpp

```diff
@@ -0,0 +1,88 @@
+#include <boost/json.hpp>
+#include <boost/json/src.hpp>
+#include <boost/program_options.hpp>
+#include <expected>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "block_generator.hpp"
```

Comment:
My previous question is still actual: does the full path works?

```cpp
#include "zkevm_framework/block_generator/block_generator.hpp"
```

Is it okay that we can use short path here or is it a bug?

Reply:
I would not call it a bug, but fixed to make it in the same style as `data_types`.